api/api/spec/spec.yaml

```diff
@@ -16857,6 +16880,47 @@ paths:
         '429':
           $ref: '#/components/responses/TooManyRequestsResponse'
 
+  /vulnerability/{agent_id}/summary/{field}:
+    get:
+      tags:
+        - Vulnerability
+      summary: "Get agent vulnerabilities' field summary"
+      description: "Return a summary of the vulnerabilities' field of an agent"
+      operationId: api.controllers.vulnerability_controller.get_vulnerabilities_field_summary
+      x-rbac-actions:
+        - $ref: '#/x-rbac-catalog/actions/vulnerability:read'
+      parameters:
+        - $ref: '#/components/parameters/pretty'
+        - $ref: '#/components/parameters/wait_for_complete'
+        - $ref: '#/components/parameters/agent_id'
+        - $ref: '#/components/parameters/vulnerability_field'
+        - $ref: '#/components/parameters/limit'
+      responses:
+        '200':
+          description: "Get agent vulnerabilities' field summary"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponse'
+              example:
+                data:
+                  severity:
+                    critical: 8
```

Comment:
I believe the response is always sorted descending, and so should be the example.